trade-finance-manager-ui/styles/layout.scss

```diff
@@ -58,13 +58,15 @@
     }
 }
 
-.ukef-with-separators {
-  > *:not(:last-child) {
-    border-bottom: 1px solid $govuk-border-colour;
+.ukef-stack {
+  &.ukef-stack-divider {
+    > *:not(:last-child) {
+      border-bottom: 1px solid $govuk-border-colour;
+    }
   }
 
   @for $spacing-unit from 1 through 9 {
-    &.ukef-separator-spacing-#{$spacing-unit} {
+    &.ukef-stack-spacing-#{$spacing-unit} {
       > *:not(:first-child) {
         @include govuk-responsive-padding($spacing-unit, 'top');
       }
```

Comment:
"stack" seems to be a more common name for a component that adds spacing and dividers between its children (e.g. [MUI Stack](https://mui.com/material-ui/react-stack/) and [Bootstrap Stack](https://react-bootstrap.netlify.app/docs/layout/stack/))